pnpm-lock.yaml

```diff
@@ -14653,4 +14604,8 @@ packages:
       - encoding
       - supports-color
     dev: true
-    patched: true
+
+patchedDependencies:
+  pkg@5.7.0:
+    hash: pp6fkuhwkrqq7cjcj7uqpf37e4
+    path: patches/pkg@5.7.0.patch
```

Comment:
do you use the latest pnpm? The lockfile doesn't look right

Reply:
It seems I was using a previous version of pnpm to install (7.1.7) but after upgrading to the latest version (7.5.0) & rerunning pnpm install, the pnpm-lock.yaml file is unchanged - is there something else I missed?